sidekiq/README.md

```diff
@@ -57,27 +57,10 @@ No additional installation is needed on your server.
        mappings:
          - match: "sidekiq.sidekiq.*"
            name: "sidekiq.*"
-         - match: "sidekiq.jobs.*.perform.avg"
-           name: "sidekiq.jobs.perform.avg"
+         - match: "sidekiq.jobs.*.perform"
+           name: "sidekiq.jobs.perform"
            tags:
              worker: "$1"
-         - match: "sidekiq.jobs.*.perform.count"
-           name: "sidekiq.jobs.perform.count"
-           tags:
-             worker: "$1"
-         - match: "sidekiq.jobs.*.perform.max"
-           name: "sidekiq.jobs.perform.max"
-           tags:
-             worker: "$1"
-         - match: "sidekiq.jobs.*.perform.median"
-           name: "sidekiq.jobs.perform.median"
-           tags:
-             worker: "$1"
-         - match: "sidekiq.jobs.*.perform.95percentile"
-           name: "sidekiq.jobs.perform.95percentile"
-           tags:
-             worker: "$1"
-
```

Comment:
It seems rules for matching those metrics are missing:

```
sidekiq.jobs.count,count,,,,Count of Sidekiq jobs,0,sidekiq,jobs count
sidekiq.jobs.success,count,,,,Count of successful Sidekiq jobs,0,sidekiq,jobs success
sidekiq.jobs.failure,count,,,,Count of failed Sidekiq jobs,-1,sidekiq,jobs failure
```

Reply:
I don't think we need rules for those (`jobs.count`, `jobs.success`, `jobs.failure`) as we don't need transform them but we need rules for:

* jobs.<job>.count => counter
* jobs.<job>.success => counter
* jobs.<job>.failure => counter
* jobs.<job>.perform

Soruce: https://github.com/mperham/sidekiq/wiki/Pro-Metrics#metrics

Reply:
Yes, right. My initial message wasn't clear. I was referring to those:

```
jobs.Foo.BarWorker.count => counter
jobs.Foo.BarWorker.success => counter
jobs.Foo.BarWorker.failure => counter
jobs.Foo.BarWorker.perform => gauge (time)
```

> Although then maybe to avoid duplication `jobs.count`, `jobs.success` and `jobs.failure` should get `.total` appended 🤔 

+1

Reply:
opted for having `.jobs` in the worker ones: https://github.com/DataDog/integrations-core/pull/6697/files#diff-2c9bcc1f9550ef40d5cc6d48658c7e7cR66